backend/src/packages/articles/libs/helpers/get-where-author-id-query.helper.ts

```diff
@@ -0,0 +1,13 @@
+import { type QueryBuilder } from 'objection';
+
+import { type ArticleModel } from '../../article.model.js';
+
+const getWhereAuthorIdQuery = (authorId?: number | null) => {
+  return (builder: QueryBuilder<ArticleModel, ArticleModel[]>): void => {
+    if (authorId && authorId > 0) {
```

Comment:
Let's better check in validation that filtering option is positive
```suggestion
    if (authorId) {
```